cli/medperf/commands/profile.py

```diff
@@ -0,0 +1,111 @@
+import os
+import typer
+import configparser
+
+from medperf import config
+from medperf.decorators import configurable
+from medperf.utils import dict_pretty_print
+from medperf.exceptions import InvalidArgumentError
+
+app = typer.Typer()
+
+
+def read_config():
+    config_p = configparser.ConfigParser()
+    config_path = os.path.join(config.storage, config.config_path)
+    config_p.read(config_path)
+    return config_p
+
+
+def write_config(config_p: configparser.ConfigParser):
+    config_path = os.path.join(config.storage, config.config_path)
+    with open(config_path, "w") as f:
+        config_p.write(f)
+
+
+@app.command("active")
+def active(profile: str):
+    """Assigns the active profile, which is used by default
+
+    Args:
+        profile (str): Name of the profile to be used.
+    """
+    config_p = read_config()
+
+    if profile not in config_p:
+        raise InvalidArgumentError("The provided profile does not exists")
+
+    config_p["active"]["profile"] = profile
+    write_config(config_p)
+
+
+@app.command("create")
+@configurable()
+def create(
+    ctx: typer.Context,
+    name: str = typer.Option(..., "--name", "-n", help="Profile's name"),
+):
+    """Creates a new profile for managing and customizing configuration
+    """
+    args = ctx.config_dict
+    config_p = read_config()
+
+    if name in config_p:
+        raise InvalidArgumentError("A profile with the same name already exists")
+
+    config_p[name] = args
+    write_config(config_p)
+
+
+@app.command("set")
+@configurable()
+def set_args(ctx: typer.Context):
+    """Assign key-value configuration pairs to the current profile.
+    """
+    profile = config.profile
+    args = ctx.config_dict
+    config_p = read_config()
+
+    current_config = config_p[profile]
+    current_config.update(args)
+    config_p[profile] = current_config
+    write_config(config_p)
+
+
+@app.command("unset")
```

Comment:
Suggestion: remove or rename the unset command. The users don't actually think about it as 'setting and unsetting'; they are just changing parameters. `unset` actually `reverts to default`, but it appears like you are 'removing' a config param.
We can change its name (reset, revert), or just remove the command. A user that uses `unset` should be aware of what is the default they are reverting to.. They can simply use `set` again. Let me know what you prefer.

I usually refer to how `gcloud` handles this stuff. Looking at https://cloud.google.com/sdk/gcloud/reference/config/unset
The unset actually makes the parameter empty, not resetting it to a default value. They have a notion of an empty parameter that, when needed, the command fails and asks the user to set it.